src/main/java/org/jenkinsci/plugins/envinject/EnvInjectBuilder$1.java

```diff
@@ -19,5 +27,8 @@
     public EnvInjectBuilder$1(@Nonnull AbstractBuild build, @CheckForNull Map<String, String> envMap) {
         super(build, envMap);
     }
-
+    
+    public EnvInjectBuilder$1(@CheckForNull Map<String, String> envMap) {
```

Comment:
I suppose it has been created for the data migration in anonymous class long-long ago. The class is deprecated/restricted, but it will stay in the codebase for a while